.github/workflows/test-and-tag.yml

```diff
@@ -20,18 +20,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: conda-incubator/setup-miniconda@v2.0.1
+      - uses: conda-incubator/setup-miniconda@v2
         with:
           mamba-version: "*"
-          channels: conda-forge,nodefaults
-          channel-priority: true
           python-version: ${{ matrix.python-version }}
           activate-environment: hyp3-sdk
           environment-file: environment.yml
 
       - name: Pytest in conda environment
         shell: bash -l {0}
         run: |
+          python -m pip install .[develop]
```

Comment:
We could invoke `postBuild` here, but probably leave it as-is, since we might eventually add other Binder-specific commands to `postBuild`?